spec/Vehicle/Vehicle.vspec

```diff
@@ -370,15 +370,15 @@ CurrentLocation.Latitude:
   min: -90
   max: 90
   unit: degrees
-  description: Current latitude of vehicle according to WGS 84.
+  description: Current latitude of vehicle according to WGS 84, as measured at position of GNSS receiver antenna.
```

Comment:
Perhaps:

as measured at the position of the GNSS receiver antenna

or

as measured by the GNSS receiver antenna

or

as measured at the GNSS receiver antenna

Reply:
Taking first variant for now, thanks!